geomstats/backend/__init__.py

```diff
@@ -7,7 +7,7 @@
 
 else:
     _backend = _default_backend
-
+    os.environ['GEOMSTATS_BACKEND'] = _backend
```

Comment:
Nice! This has been changed by a recent PR though, so you need to rebase on master